### PR TITLE
[C# sample] 03.adaptiveCards.a.typeAheadBot

### PR DESCRIPTION
**Pull request by kuojianlu**
Ready for review, but still have 2 problems needed to be solved:
- **[Solved]** Hit a bug when building this sample, see https://github.com/microsoft/teams-ai/issues/326 for the details.
- After selecting an item from the choice set, Adaptive Card shows "There was an error" and the "Submit" button not working. Same error happens in [typeahead sample in Microsoft-Teams-Samples ](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-type-ahead-search-adaptive-cards/csharp). Created an issue for track: https://github.com/microsoft/teams-ai/issues/333.

### CODE REVIEW

File: dotnet/samples/03.adaptiveCards.a.typeAheadBot/teamsapp.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      # will skip the resource group selection prompt.
+      resourceGroupName: ${{AZURE_RESOURCE_GROUP_NAME}}
+      templates:
+        - path: ./infra/azure.bicep # Relative path to this file
+          # Relative path to this yaml file.
+          # Placeholders will be replaced with corresponding environment
+          # variable before ARM deployment.
+          parameters: ./infra/azure.parameters.json
+          # Required when deploying ARM template
+          deploymentName: Create-resources-for-tab
```
Comment (swatDong) [inline dotnet/samples/03.adaptiveCards.a.typeAheadBot/teamsapp.yml:57]: better deploy name

Comment (kuojianlu) [reply dotnet/samples/03.adaptiveCards.a.typeAheadBot/teamsapp.yml]: Nice catch. Updated.

**Review by swatDong — APPROVED** [review]
Code looks good to me.
For the choice set selection issue. Create a bug/issue so we can track it's caused by SDK/Bot Framework/Teams?

**Comment by kuojianlu** [comment]
> Code looks good to me. For the choice set selection issue. Create a bug/issue so we can track it's caused by SDK/Bot Framework/Teams?

Created an issue: https://github.com/microsoft/teams-ai/issues/333. I'm still not sure of the cause.

**Review by singhk97 — APPROVED** [review]

**Review by singhk97 — APPROVED** [review]

**Pull request merged**